### ISSUE TITLE
Installing additional OS

### ISSUE THREAD

**Issue by icdmkg2** [kasmtech/workspaces-issues#100]
Hello there Can anyone tell me if there is a way to install different operating system etc Windows, Debian, ArchLinux

**Comment by j-travis** [comment]
Our officially supported distros/versions can be found here - Debian is one of them: https://kasmweb.com/docs/latest/install/system_requirements.html#operating-system

However, at the end of the day the main system requirements are openssl, docker >18.06 and docker compose v2. If the installer detects those it will continue and try to load up system: 
https://kasmweb.com/docs/latest/install/system_requirements.html#docker-and-docker-compose

Here is a summary and example on installing on an unsorted OS: https://kasmweb.com/docs/latest/how_to/other_operating_systems.html

See here: for installing docker compose v2
https://github.com/kasmtech/workspaces-issues/issues/93#issuecomment-1015275365

**Comment by icdmkg2** [comment]
No i meant installing Operating System's on kasm

**Comment by j-travis** [comment]
Oh I see.
Creating a new base image is very complex - so the answer is likely not.

If you are a solid understanding of linux and docker you can review our "core" images and try creating your own. 

You won't be able to create a Windows version but you may be able to create a linux version. There are a lot of dependencies though.


https://github.com/kasmtech/workspaces-core-images
https://kasmweb.com/docs/latest/how_to/building_images.html

**Comment by icdmkg2** [comment]
k thanks

**Comment by icdmkg2** [comment]
Will there be windows 10 support for kasm ?

**Comment by j-travis** [comment]
Likely not.
You cant run graphical application inside windows containers so until that is a possibility we can't provide a windows solution as long as Kasm is 100% containerized

**Comment by kaosproject** [comment]
Hello is it possible to install a additional OS on the standard offer ? like Kali ? 
thanks

**Comment by icdmkg2** [comment]
Yes you can install Kali Network Chuck showed how to enable kali

**Comment by kaosproject** [comment]
> Yes you can install Kali Network Chuck showed how to enable kali

I use the cloud of Kasm not mine I don't have acces to the administration tools

**Comment by icdmkg2** [comment]
So if you want Kali you need to tell your admin to enable it from the images tab at the administration panel

**Comment by kaosproject** [comment]
My admin is Kasm I'm on the Kasm cloud

**Comment by j-travis** [comment]
@kaosproject , unfortunately we do not offer Kali Linux on the cloud subscription offering.

If you install workspaces yourself (on prem or on the cloud) then you can use our Kali Image. It's installed by default on amd64 systems you just have to enable it from the Admin -> Image page.  An arm64 image for Kali is not available at this time.

**Comment by icdmkg2** [comment]
Bruh you do support kali because I have kali installed and running

**Comment by mikev1963** [comment]
Is it possible to have ParrotOS ?

**Comment by pmisner** [comment]
Just curious how running windows on KVM in a Linux instance would work.

**Comment by teja-kasm** [comment]
We’re currently enhancing our GitHub issue tracker to improve organization and efficiency.

As part of this process, we’re closing older issues to help keep the tracker focused on current and active topics.

If this issue is still relevant or you’re still experiencing the problem, please feel free to create a new issue.

Thank you for your understanding!
